modules/nw-ovn-kubernetes-change-log-levels.adoc

```diff
@@ -86,16 +86,23 @@ configmap/env-overrides.yaml created
 [source,terminal]
 ----
 $ oc delete pod -n openshift-ovn-kubernetes \
---field-selector spec.nodeName=ip-10-0-217-114.ec2.internal -l app=ovnkube-node
+--field-selector spec.nodeName=ci-ln-3njdr9b-72292-5nwkp-master-0 -l app=ovnkube-node
 ----
 +
 [source,terminal]
 ----
 $ oc delete pod -n openshift-ovn-kubernetes \
---field-selector spec.nodeName=ip-10-0-209-180.ec2.internal -l app=ovnkube-node
+--field-selector spec.nodeName=ci-ln-3njdr9b-72292-5nwkp-master-2 -l app=ovnkube-node
 ----
 +
 [source,terminal]
 ----
-$ oc delete pod -n openshift-ovn-kubernetes -l app=ovnkube-master
+$ oc delete pod -n openshift-ovn-kubernetes -l app=ovnkube-node
+----
+
+. Verify that the log levels have been changed by running the following command:
++
+[source,terminal]
+----
+$ oc logs -n openshift-ovn-kubernetes <_pod_name_>
```

Comment:
You need to specify the pod, otherwise it defaults.

For node specific:
For `OVN_KUBE_LOG_LEVEL` check `ovnkube-controller` using  `-c ovnkube-controller`
For `OVN_LOG_LEVEL` check `ovn-controller`

For `_master`:
For `OVN_LOG_LEVEL` check `northd`, `nbdb`, `sbdb`

Or just use `--all-containers`.

This should dump everything.
```
oc logs -n openshift-ovn-kubernetes --all-containers --prefix ovnkube-node-2cpjc | grep -E -m 10 '(Logging config:|vconsole|DBG)'
```

Reply:
```sh
oc logs -n openshift-ovn-kubernetes --all-containers --prefix ovnkube-node-2cpjc | grep -E -m 10 '(Logging config:|vconsole|DBG)'
[pod/ovnkube-node-2cpjc/sbdb] + exec /usr/share/ovn/scripts/ovn-ctl --no-monitor '--ovn-sb-log=-vconsole:info -vfile:off -vPATTERN:console:%D{%Y-%m-%dT%H:%M:%S.###Z}|%05N|%c%T|%p|%m' run_sb_ovsdb
[pod/ovnkube-node-2cpjc/ovnkube-controller] I1012 14:39:59.984506   35767 config.go:2247] Logging config: {File: CNIFile:/var/log/ovn-kubernetes/ovn-k8s-cni-overlay.log LibovsdbFile:/var/log/ovnkube/libovsdb.log Level:5 LogFileMaxSize:100 LogFileMaxBackups:5 LogFileMaxAge:0 ACLLoggingRateLimit:20}
[pod/ovnkube-node-2cpjc/northd] + exec ovn-northd --no-chdir -vconsole:info -vfile:off '-vPATTERN:console:%D{%Y-%m-%dT%H:%M:%S.###Z}|%05N|%c%T|%p|%m' --pidfile /var/run/ovn/ovn-northd.pid --n-threads=1
[pod/ovnkube-node-2cpjc/nbdb] + exec /usr/share/ovn/scripts/ovn-ctl --no-monitor '--ovn-nb-log=-vconsole:info -vfile:off -vPATTERN:console:%D{%Y-%m-%dT%H:%M:%S.###Z}|%05N|%c%T|%p|%m' run_nb_ovsdb
[pod/ovnkube-node-2cpjc/ovn-controller] 2023-10-12T14:39:54.552Z|00002|hmap|DBG|lib/shash.c:114: 1 bucket with 6+ nodes, including 1 bucket with 6 nodes (32 nodes total across 32 buckets)
[pod/ovnkube-node-2cpjc/ovn-controller] 2023-10-12T14:39:54.553Z|00003|hmap|DBG|lib/shash.c:114: 1 bucket with 6+ nodes, including 1 bucket with 6 nodes (64 nodes total across 64 buckets)
[pod/ovnkube-node-2cpjc/ovn-controller] 2023-10-12T14:39:54.553Z|00004|hmap|DBG|lib/shash.c:114: 1 bucket with 6+ nodes, including 1 bucket with 7 nodes (32 nodes total across 32 buckets)
[pod/ovnkube-node-2cpjc/ovn-controller] 2023-10-12T14:39:54.553Z|00005|reconnect|DBG|unix:/var/run/openvswitch/db.sock: entering BACKOFF
[pod/ovnkube-node-2cpjc/ovn-controller] 2023-10-12T14:39:54.553Z|00007|reconnect|DBG|unix:/var/run/openvswitch/db.sock: entering CONNECTING
[pod/ovnkube-node-2cpjc/ovn-controller] 2023-10-12T14:39:54.553Z|00008|ovsdb_cs|DBG|unix:/var/run/openvswitch/db.sock: SERVER_SCHEMA_REQUESTED -> SERVER_SCHEMA_REQUESTED at lib/ovsdb-cs.c:423
```